### ISSUE TITLE
With regards to a perceptual loss 

### ISSUE THREAD

**Issue by PetrochukM** [magenta/ddsp#12]
Thanks for the work you've done, this work is very exciting because it's intuitive to me! I do have a question though...

It is encouraged to use the multi-resolution spectrogram loss; however, the spectrogram loss does not incorporate a number of perceptual biases:

- The MEL Scale
- A-Frequency weighting
- Decibel scale

How come?

Furthermore, in the paper, it mentions computing the reconstruction loss without the log scale. Given that human perception is non-linear, this choice doesn't make sense to me. Why would you compute the loss without the log scale?

**Comment by jesseengel** [comment]
Hi, thanks for the questions! Better perceptual losses is certainly an area for further investigation. 

First off, as you can see from the code:
https://github.com/magenta/ddsp/blob/master/ddsp/losses.py

the standard autoencoder configuration
https://github.com/magenta/ddsp/blob/master/ddsp/training/gin/models/ae.gin#L41
actually does use a log-amplitude scaled spectrogram (dB scale). 

The configuration that jointly learns to predict frequency
https://github.com/magenta/ddsp/blob/master/ddsp/training/gin/models/ae_abs.gin
uses mel-frequency scale as input to the encoder, and addtionally uses a pretrained CREPE network to capture higher level audio statistics, similar to "perceptual losses" in the image literature. 

All models use A-Frequency weighting for calculation of "loudness_db" which is used as a conditioning feature.

There are certainly more options for losses inspired by psychophysics (including several even in `losses.spectral_loss()`, but we simply haven't used them yet because they haven't yet been necessary. 

Feel free to try them out and if you find one that clearly works better, write it up and contribute it to the library.

**Comment by PetrochukM** [comment]
Thanks for the response. 

With regards to the autoencoder configuration...
https://github.com/magenta/ddsp/blob/cd9811675873dc3f5c5fb3a51b5a48923b3330ae/ddsp/training/gin/models/ae.gin#L40

Is there a benefit to training on a none-log-amplitude scaled spectrogram? It looks like it's also used in the loss.
